__tests__/__fixtures__/requests/multipart-data.js

```diff
@@ -22,9 +22,13 @@ module.exports = {
               {
                 name: 'foo',
                 value: 'Hello World',
-                fileName: 'hello.txt',
+                fileName: '__tests__/__fixtures__/files/hello.txt',
```

Comment:
Changed this to map to where `hello.txt` is in this repo so tests that run this fixture will be able to locate and upload the file.